### PR TITLE
NEP: NeoContract ABI

### PR DESCRIPTION
**Pull request by erikzhang**
It should be merged after #118.

### CODE REVIEW

**Comment by Tommo-L** [comment]
Should we add a specification for the `verify` method?

**Comment by erikzhang** [comment]
No. It should be included in the neo specification.

**Comment by ixje** [comment]
I think this document should describe `_deploy` and `_initialize`

File: nep-14.mediawiki [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+<code>name</code> is the name of the method, which can be any valid identifier.
+
+<code>offset</code> is the offset of this method in the script.
+
+<code>safe</code> indicates if it's safe to call this method. If a method is marked as safe, the user interface will not give any warnings when it is called by any other contract.
+
+<code>parameters</code> is an array of Parameter objects which describe the details of each parameter in the method.
+
+<code>returntype</code> indicates the return type of the method. It can be one of the following values: <code>Signature</code>, <code>Boolean</code>, <code>Integer</code>, <code>Hash160</code>, <code>Hash256</code>, <code>ByteArray</code>, <code>PublicKey</code>, <code>String</code>, <code>Array</code>, <code>Map</code>, <code>InteropInterface</code>, <code>Void</code>.
```
Comment (erikzhang) [inline nep-14.mediawiki:66]: And `Any`?

File: nep-14.mediawiki [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+<pre>
+{
+  "name": "from",
+  "type": "Hash160"
+}
+</pre>
+
+<code>name</code> is the name of the parameter, which can be any valid identifier.
+
+<code>type</code> indicates the type of the parameter. It can be one of the following values: <code>Signature</code>, <code>Boolean</code>, <code>Integer</code>, <code>Hash160</code>, <code>Hash256</code>, <code>ByteArray</code>, <code>PublicKey</code>, <code>String</code>, <code>Array</code>, <code>Map</code>, <code>InteropInterface</code>.
```
Comment (erikzhang) [inline nep-14.mediawiki:96]: And `Any`?

**Comment by erikzhang** [comment]
Merge?

**Review by shargon — DISMISSED** [review]

**Pull request merged**